lib/chat_api/accounts.ex

```diff
@@ -53,9 +54,20 @@ defmodule ChatApi.Accounts do
 
   """
   def create_account(attrs \\ %{}) do
-    %Account{}
-    |> Account.changeset(attrs)
-    |> Repo.insert()
+    result =
+      Ecto.Multi.new()
+      |> Ecto.Multi.insert(:account, Account.changeset(%Account{}, attrs))
+      |> Ecto.Multi.run(:widget_settings, fn _repo, %{account: account} ->
+        %WidgetSetting{}
+        |> WidgetSetting.changeset(%{account_id: account.id})
+        |> Repo.insert()
```

Comment:
just curious, why `Repo` here instead of the `_repo` above?

Reply:
You're right. Using the previous `repo` variable might be better for `Ecto.Multi.run` block.
I just copy-pasted it from the previously deleted `create_widget_setting`. 🙂